py/packages/genkit/src/genkit/ai/embedding.py

```diff
@@ -2,9 +2,20 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from collections.abc import Callable
+from typing import Any
 
 from pydantic import BaseModel
 
+type Embedding = list[float]
+
+
+# TODO(ssbushi): Replace with generated type
+class EmbeddingModel(BaseModel):
+    """Represents an embedding with metadata."""
+
+    embedding: Embedding
+    metadata: dict[str, Any] | None = None
+
```

Comment:
This is a placeholder for now, will update with the generated type we should get from tooling.